docs/mine-hnt/validators/validator-deployment-guide.mdx

```diff
@@ -0,0 +1,389 @@
+
+
+```
+$ helium-wallet balance
+```
+
+If all went to plan, you'll see this:
+
+```
++-----------------------------------------------------+----------------+--------------+-----------------+
+| Address                                             | Balance        | Data Credits | Security Tokens |
++-----------------------------------------------------+----------------+--------------+-----------------+
+| 1aP7nm6mGLMFtgiHQQxbPgKcBwnuQ6ehgTgTN8zjFuxByzJ8eA5 | 10005.00000000 | 0            | 0.00000000      |
++-----------------------------------------------------+----------------+--------------+-----------------+
+```
+Glorious. You have the `10000` TNT to stake one validator, and a bit extra to cover the transaction fees. Now it's time to get your Validator running. 
+
+
+## Run a Validator 
+
+With TNT in hand, it's time to deploy and run the actual Validator (referred to here as `miner`). 
+
+:::info
+
+If you haven't already, please review the [Validator Technical Requirements](/mine-hnt/validators/requirements)
+
+:::
+
+
+You have two options for deploying the Validator. Choose your own adventure:
+
+* [Deploy the Miner Using Docker](#deploy-the-miner-using-docker) 
+* [Build the Miner form Source](#deploy-miner-from-source)
+
+
+
+### Deploy the Miner Using Docker 
+
+Start by updating your packager manager registry:
+
+```
+sudo apt-get update
+```
+
+And install Docker itself:
+
+```
+sudo apt-get install docker.io
+```
+
+To avoid needing to use Docker with `sudo` privileges, add your user to the docker group, replacing $USER with your username here:
+
+```
+sudo usermod -aG docker $USER
+```
+
+Log in and out of your account to apply these changes. You are now ready to use Docker.
+
+:::warning
+
+On amd64 systems `AVX` support is required. Verify that it exists on your host system by running the following:
+
+```
+grep avx /proc/cpuinfo
+grep avx2 /proc/cpuinfo
+```
+
+If nothing is returned from these commands, your host system does not have AVX support and your Validator may not be stable.
+
+:::
+
+#### Run a Docker Container
+
+Before running the container for the first time, it is advisable to pick a "system mount point". This allows you to easily maintain your Validator's blockchain identity - specifically it `swarm keys - and blockchain state through image updates.
+
+The validator directory should be created on a persistent EBS volume (if you're using AWS) or something similar that lives across server lifetimes.
+
+```
+mkdir $HOME/validator_data; source=$HOME/validator_data
+```
+
+You can then use the `run` command to start your container for the first time:
+
+```
+docker run -d \
+--restart always \
+--publish 2154:2154/tcp \
+--name validator \
+--mount type=bind,source=$HOME/validator_data,target=/var/data \
+quay.io/team-helium/validator:miner-val-amd64_val0.1.7
```

Comment:
```suggestion
quay.io/team-helium/validator:miner-val-amd64_val0.1.8
```
I believe we can reliably point to 0.1.8 now.

Reply:
Or maybe just `latest`?

Reply:
Switched to `latest-val-amd64`. Thanks.